src/edge/edgeManager.ts

```diff
@@ -383,11 +386,12 @@ export class EdgeManager {
             const JobInfo: any = await saManager.getJobInfo(job);
             imageName = JobInfo.settings.image;
             moduleTwin = JobInfo.twin.content;
+            createOptions = JobInfo.settings.createOptions ? JSON.stringify(JobInfo.settings.createOptions) : "";

```

Comment:
I think we should put the string replacement logic here. The outer code can use the returned `createOptions` as a string without doing escaping/unescaping each time.